packages/medusa/src/types/pricing.ts

```diff
@@ -26,6 +28,8 @@ export type PricingContext = {
 export type ShippingOptionPricing = {
   price_incl_tax: number | null
   tax_rates: TaxServiceRate[] | null
+  price_includes_tax?: boolean
```

Comment:
nit: this results in some slightly confusing responses in the API

```js
{
  "id": "so_01FX85781B265SXE4T7ESQHHNQ",
  "name": "UPS Home Delivery (2-3 business days)",
  "amount": 1200,
  "price_incl_tax": 1200,
  "tax_rates": [{ "rate": 25, ... }],
  "tax_amount": 240,
  "includes_tax": true, // <-- This will always be the same as "price_includes_tax"
  "price_includes_tax": true, // <-- This will always be the same as "includes_tax"
  ...
}
```

Think we should clean up the response to only spit back `includes_tax`

Reply:
I believe we should be able to omit `price_includes_tax` entirely from the type as it is currently not being used. We add it [here](https://github.com/medusajs/medusa/blob/5a2407f48229ef249db1d2488acce0198d5f478c/packages/medusa/src/services/pricing.ts#L484) with the same value as `includes_tax`. I might have missed some context - let me know @pKorsholm

Reply:
I think that you are right, since the input is the shipping options it means that the consumer have also access to it and therefore has access to `shippingOption.includes_tax`. we can safely remove that property as it is a duplicate of the one from the entity itself 👍